dependency-check-suppressions.xml

```diff
@@ -4,11 +4,9 @@
         CVE-2021-37533 refer [Ticket]
         CVE-2021-4277 refer [Ticket]
```

Comment:
Can you please add the missing jira references for other CVEs too?

Reply:
Links Added